src/migrate.js

```diff
@@ -121,14 +122,18 @@ function filterUnappliedMigrations(orderedMigrations) {
   }
 }
 
+const isFile = (type) => (fileName) => fileName.endsWith(type)
+const isSqlFile = isFile(".sql")
+const isJsFile = isFile(".js")
 const readDir = bluebird.promisify(fs.readdir)
 function loadMigrationFiles(directory, log) {
   log(`Loading migrations from: ${directory}`)
   return readDir(directory)
     .then((fileNames) => {
       log(`Found migration files: ${fileNames}`)
       return fileNames
-        .filter((fileName) => fileName.toLowerCase().endsWith(".sql"))
+        .map((fileName) => fileName.toLowerCase())
```

Comment:
`toLowerCase` can't be in the map here - we only want to lower case it to check the extension. Doing it in the map will cause us to try to load the files with lower case names, which will fail if any of them have upper case letters.

Might be worth having some test migration files with uppercase letters to flush this sort of thing out.